requirements.txt

```diff
@@ -1,3 +1,4 @@
 django==2.1
 psycopg2
 whitenoise
+pillow
```

Comment:
If pillow is in a requirements.txt file in the Docker folder, do we also need it in the main project?

Reply:
Because pillow is used by our test algorithm (think of it as a system test) it is a requirement of the project however I'd say its more of a dev-requirement (@eetar1) than an actual requirement (as it is for testing purposes, ie not required for the cloud-platform to run.)

Reply:
Addressed in 88a8fba